app/src/main/java/com/x8bit/bitwarden/data/platform/util/ServerVersionUtils.kt

```diff
@@ -0,0 +1,45 @@
+package com.x8bit.bitwarden.data.platform.util
+
+import com.x8bit.bitwarden.data.platform.datasource.disk.model.ServerConfig
+import kotlin.text.split
+import kotlin.text.toIntOrNull
+
+private const val VERSION_SEPARATOR = "."
+private const val SUFFIX_SEPARATOR = "-"
+
+/**
+ * Checks if the server version is greater than another provided version, returns true if it is.
+ */
+ fun isServerVersionAtLeast(serverConfig: ServerConfig?, version: String): Boolean {
+    val serverVersion = serverConfig
+        ?.serverData
+        ?.version
+
+    val serverVersionParts = getVersionComponents(serverVersion)
+    val otherVersionParts = getVersionComponents(version)
+
+    if (serverVersionParts == null || otherVersionParts == null) {
+        return false
+    }
+
+    for (i in serverVersionParts.indices) {
+        val serverPart = serverVersionParts.getOrNull(i)?.toIntOrNull() ?: 0
+        val otherPart = otherVersionParts.getOrNull(i)?.toIntOrNull() ?: 0
+
+        if (serverPart > otherPart) {
+            return true
+        } else if (serverPart < otherPart) {
+            return false
+        }
+    }
+
+    return true // Versions are equal
```

Comment:
Comment should be above the return not on the same line.

Can we also add a test for this case